### DISCUSSION TITLE
Hyprdots discord

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#1149] [General]
Who among here are interested on making a Hyprdots discord channel? I don't have experience on managing a channel.It would be cool if someone can help me make this possible.

**Comment by opticblu** [comment]
I'd help, you just need some mods basically?

In general, ngl I don't love discord, but it's where the actual hyprland (like dev vaxry hyprland) chat group is, soooo makes sense

in favor of this, would be able to help (not a whole lot, lots of irl/work/family obligations, but if you need a 'backstop' or competent-ish mod, happy to) -- actually searched for "discord" in this repo to see if there already was one after replying to an open issue on swww

**Reply by kRHYME7** [reply]
Here's my discord ``` khing0v0 ``` 😁

**Reply by opticblu** [reply]
yelloducky is mine, sending request, about to turn in for the night (USA) though so may not see messages until tomorrow

**Comment by se7en-x230** [comment]
https://matrix.org/ecosystem/bridges/

I would very much appreciate a matrix bridge
